lib/std/debug.zig

```diff
@@ -1390,8 +1390,9 @@ pub const DwarfInfo = struct {
 
                     // All the addresses in the list are relative to the value
                     // specified by DW_AT_low_pc or to some other value encoded
-                    // in the list itself
-                    var base_address = try compile_unit.die.getAttrAddr(DW.AT_low_pc);
+                    // in the list itself.
+                    // If no starting value is specified use zero.
+                    var base_address = compile_unit.die.getAttrAddr(DW.AT_low_pc) catch 0;
```

Comment:
this should probably do the `catch |err| switch (err)` pattern. I'm guessing this behavior is correct for both `error.InvalidDebugInfo` and `error.MissingDebugInfo`. But if, for example, `error.OutOfMemory` was added to the set, one would want to bubble that one up rather than using `0`.

Reply:
Good catch, sometimes a `catch` is so tempting that you don't stop and think about this kind of details.